src/app/zones/views/ZoneListView.vue

```diff
@@ -33,6 +33,7 @@
         <DataSource
           v-slot="{ data, error, refresh }: ZoneOverviewCollectionSource"
           :src="`/zone-cps?size=${props.size}&page=${props.page}`"
+          @change="setIsCreateZoneButtonVisible"
```

Comment:
I don't get why the button doesn't move in the v-else and inside this DataSource.
This fixes:

- need to repeat checks to `store.getters`
- need to add global state
- readability (the component is poorly named so I had to check the code to see that the `MultizoneInfo` component is actually the component that shows that we're in standalone).

I've just looked and zoneIngressList does this and it's a lot easier to follow.
Btw I also just noticed that the conditional is missing from zoneEgressList

Reply:
This button, by design, is placed in the same row as the heading "Zone Control Planes". To do this, we use `AppView`’s `actions` slot. That has the consequence that the button can’t move further into the tree.

The next alternative is moving the `DataSource` further out (specifically around `AppView`) so its data can be used for this element without setting the `isCreateZoneButtonVisible` state on the component. That would however cause the `DataSource` component to trigger a request without the check attached to the `MultizoneInfo` component preventing it in standalone mode. We could instruct `DataSource` to only make the request conditionally (using its `shouldMakeRequest` prop) to address this.

However, it would be easier to drop the `MultizoneInfo` component instead. We don’t show the Zone & Ingress views in standalone mode (though you can access them via URL) so its information is unlikely to be seen by users anyway. Then, moving the `DataSource` out around `AppView` provides the right things with the necessary data access.

Reply:
Because we want to show:

- if no Zones: empty state with create button
- if Zones: create button above

(I.e. following what we do in overview)

Reply:
You could use the `if=getMultiCluster` in the datasource no? In any case it doesn't matter too much.